packages/@lwc/features/src/runtime.ts

```diff
@@ -5,17 +5,22 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 import features, { FeatureFlagLookup, FeatureFlagValue } from './flags';
-import { assert, create, isFalse, isTrue, isUndefined } from '@lwc/shared';
+import { create, isFalse, isTrue, isUndefined } from '@lwc/shared';
 
 const runtimeFlags: FeatureFlagLookup = create(null);
 
 // This function is not whitelisted for use within components and is meant for
 // configuring runtime feature flags during app initialization.
 function setFeatureFlag(name: string, value: FeatureFlagValue) {
-    assert.invariant(
-        isTrue(value) || isFalse(value),
-        `Invalid ${typeof value} value specified for the "${name}" flag. Runtime feature flags can only be set to a boolean value.`
-    );
+    const isBoolean = isTrue(value) || isFalse(value);
+    if (!isBoolean) {
+        const message = `Invalid ${typeof value} value specified for the "${name}" flag. Runtime feature flags can only be set to a boolean value.`;
```

Comment:
we really need to figure out what are we doing to do with error codes... this message should only show in dev, and in prod a numeric value, but that's just a general comment, no changes needed here...

Reply:
and personally, i prefer to fail in dev mode (the invariant but wrapped in the `if (process.env.NODE_ENV !== 'production')`).

Reply:
We throw a `TypeError` in the else-block so the outcome will be the same. Are you saying you prefer using the assertion library?

Reply:
no, well, im more used to the assertions, but i meant that: i prefer to fail in dev mode **only**, but like this is fine.